hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListener.java

```diff
@@ -231,6 +190,57 @@ public void onItem(RecordItem recordItem) throws ImporterException {
         log.debug("Storing transaction: {}", transaction);
     }
 
+    // reduce Cognitive Complexity of onItem() by moving the "dispatcher" part of the code into a separate method.
+    // This still has Cognitive Complexity of 19, but it really doesn't make sense to split into two parts.
+    @SuppressWarnings("java:S3776")
+    private void processTransaction(RecordItem recordItem) {
+        TransactionRecord txRecord = recordItem.getTransactionRecord();
+        int transactionTypeValue = recordItem.getTransactionType();
+        long consensusTimestamp = DomainUtils.timeStampInNanos(txRecord.getConsensusTimestamp());
```

Comment:
It's already parsed.

```suggestion
        long consensusTimestamp = recordItem.getConsensusTimestamp());
```

Reply:
Done.